libbeat/docs/overview.asciidoc

```diff
@@ -4,22 +4,34 @@
 <titleabbrev>Beats overview</titleabbrev>
 ++++
 
-The _Beats_ are open source data shippers that you install as _agents_ on
-your servers to send different types of operational data to
-https://www.elastic.co/products/elasticsearch[Elasticsearch]. Beats can
-send data directly to Elasticsearch or send it to Elasticsearch via Logstash, which
-you can use to parse and transform the data.
-
-_Packetbeat_, _Filebeat_, _Metricbeat_, and _Winlogbeat_ are a few examples of _Beats_. Packetbeat
-is a network packet analyzer that ships information about the transactions
-exchanged between your application servers. Filebeat ships log files from your servers. Metricbeat is a server monitoring agent
-that periodically collects metrics from the operating systems and services
-running on your servers. And Winlogbeat ships Windows event logs.
+{beats} are open source data shippers that you install as agents on your
+servers to send operational data to
+https://www.elastic.co/products/elasticsearch[{es}]. Elastic provides {beats}
+for capturing:
+
+[horizontal]
+Network traffic:: https://www.elastic.co/products/beats/packetbeat[Packetbeat] 
+Log files:: https://www.elastic.co/products/beats/filebeat[Filebeat] 
+Metrics:: https://www.elastic.co/products/beats/metricbeat[Metricbeat] 
+Windows event logs:: https://www.elastic.co/products/beats/winlogbeat[Winlogbeat] 
+Audit data:: https://www.elastic.co/products/beats/auditbeat[Auditbeat] 
+Uptime:: https://www.elastic.co/products/beats/heartbeat[Heartbeat] 
```

Comment:
I think "Availability" is a more accurate term to use for Heartbeat than "Uptime". Availability is whether or not the service is reachable. It's possible to have 100% uptime but be unreachable if the network is down. And Heartbeat will tell if you if it could connect to the service (hence telling you if it was available).

Reply:
Good point. I wonder if we need to change this on the product pages, too, because we describe Heartbeat as providing uptime monitoring.